.travis.yml

```diff
@@ -15,6 +16,7 @@ env:
   - VERSION=9.3-2.3 VARIANT=alpine
   - VERSION=9.2-2.3
   - VERSION=9.2-2.3 VARIANT=alpine
+  - VERSION=10-2.4
```

Comment:
What about Alpine variants?

Reply:
I'm taking a stab at updating this myself (assuming you didn't uncheck the checkbox that lets me push to your branch).

Reply:
Great, `Allow edits from maintainers` is still checked